learn/what_is_meilisearch/comparison_to_alternatives.md

```diff
@@ -9,12 +9,144 @@
+|---|:---:|:----:|:---:|:---:|
+| Typo tolerant  | ✅ | ✅ | ✅ | 🔶 <br>Need to be specified by fuzzy queries. |
+| Orderable ranking rules | ✅ | ✅ | 🔶 <br>Tie-breaking order is limited by a unique scoring rule. | ❌|
+| Custom rules | ✅ | ✅ | 🔶 <br>Limited to one default sorting rule. | 🔶 <br>Function score query
+| Query field weights | ✅ | ✅ | ✅ | ✅ |
+| Synonyms | ✅ | ✅ | ✅ | ✅ |
+| Stop words | ✅ | ✅ | ❌ | ✅ |
+| Automatic language detection | ✅ | ✅ | ❌ | ❌ |
+| All language supports | ✅ | ✅ | ❌ <br> Only space separated | ✅ |
+
+##### Security
+
+| - | MeiliSearch | Algolia | Typesense | Elasticsearch |
+|---|:---:|:----:|:---:|:---:|
+| API Key Management | ✅ | ✅ | ✅ | ✅ |
+| Tenant tokens & multi-tenant indexes | **Q1 2022** <br> Configurable end-user hard filters per index. | ✅  <br> Hard filters are not configurable per index for an end-user tenant key.   | ✅ <br> Hard filters are not configurable per index for an end-user tenant key. | ✅ <br> Role based |
+
+##### Search
+
+| - | MeiliSearch | Algolia | Typesense | Elasticsearch |
+|---|:---:|:----:|:---:|:---:|
+| Placeholder search | ✅ | ✅ | ✅ | ✅ |
+| Multi-index search | Early 2022 | ✅ | ✅ | ✅ |
+| Exact phrase search | ✅ | ✅ | ❌ | ✅ |
+| Geo search |  ✅  | ✅ | ✅ | ✅ |
+| Sort by  |  ✅  | 🔶 <br> Limited to one `sort_by` rule per index. Index have to be duplicated for each sort field and sort order. | 🔶 <br> Do not support sort on string field. | ✅ |
+| Filtering |  ✅ <br>Support complex filter queries with an SQL-like syntax.  | 🔶 <br>Does not support `OR` operation across multiple fields. | 🔶 <br>Does not support `OR` operation across multiple fields. | ✅ |
+| Facetting |  ✅ | ✅ | ✅ | ✅ |
+| Facet value searches | ❌ | ✅ | ✅ | ❌ |
+| Distinct <br><div style="color:#A9A9A9;margin-top:0.8em;">De-duplicate documents by a field value</div>| ✅ | ✅ | ✅  | ✅ |
+| Grouping <br><div style="color:#A9A9A9;margin-top:0.8em;">Bucket documents by field values</div> | ❌ | ✅ | ✅  | ✅ |
+
+##### Vizualize
+
+| - | MeiliSearch | Algolia | Typesense | Elasticsearch |
+|---|:---:|:----:|:---:|:---:|
+| [Visual Dashboard](https://github.com/meilisearch/mini-dashboard) | ✅ | 🔶 <br>Cloud product | 🔶 <br>Cloud product | ✅ |
+
+#### Deploy
+
+| - | MeiliSearch | Algolia | Typesense | Elasticsearch |
+|---|:---:|:----:|:---:|:---:|
+| Self-hosted | ✅  | ❌  | ✅  | ✅ |
+| Official 1-click deploy | ✅ <br> [DigitalOcean](https://marketplace.digitalocean.com/apps/meilisearch) <br> [Platform.sh](https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/meilisearch/.platform.template.yaml) | ❌ | 🔶 <br>Only for the cloud-hosted solution. | ❌ |
+| Official cloud-hosted solution | [Join the beta](https://meilisearch.typeform.com/to/FtnzvZfh?typeform-source=comparative-table) | ✅ | ✅ | ✅ |
+| High availability | Expected 2022 | ✅ | ✅ | ✅ |
+| Run-time dependencies | None | n/a | None | None |
```

Comment:
What's the difference between `n/a` and `None`? If at all possible, I'd recommend making them the same otherwise the reader is likely to ask that same question and find no reply within this document.

Reply:
If `n/a` is necessary here, make sure that its spelling is consistent throughout the document (e.g. it's always either `N/A` or `n/a`).

```suggestion
| Run-time dependencies | None | N/A | None | None |
```

Reply:
> What's the difference between `n/a` and `None`? If at all possible, I'd recommend making them the same otherwise the reader is likely to ask that same question and find no reply within this document.

I don't know if it's the case for run-time dependencies, @gmourier could help, but here is the difference that I see:
`None` means zero
`N/A` means that this item can't be considered for said alternative, it's not a question that can be asked (for some reasons, like the engine's design philosophy)